docs/src/terms.md

```diff
@@ -29,16 +29,56 @@ with the `HalfInteger` type.
 
 ```@docs
 terms
-count_terms
 ```
 
-The `L` and `S` quantum numbers are not, in general, sufficient to uniquely identify a term.
-The [`IntermediateTerm`](@ref) type allows one to specify an additional quantum number which
-would uniquely identify the term.
+## Term multiplicity and intermediate terms
+
+For subshells starting with `d³`, the possible terms may occur more
+than once (multiplicity higher than one), corresponding to different
+physical states. These arise from different sequences of coupling the
+``w`` equivalent electrons of the same ``\ell``, and are distinguished
+using a _seniority number_, which the [`IntermediateTerm`](@ref) type
+implements.
 
 ```@docs
 IntermediateTerm
 intermediate_terms
+count_terms
+```
+
+AtomicLevels.jl uses the algorithm presented in
+
+- _Alternative mathematical technique to determine LS spectral terms_
+  by Xu Renjun and Dai Zhenwen, published in JPhysB, 2006.
+
+  [doi:10.1088/0953-4075/39/16/007](https://dx.doi.org/10.1088/0953-4075/39/16/007)
+
+to compute the multiplicity of individual subshells, beyond the
+trivial cases of a single electron or a filled subshell.
+
+In the following, ``S'=2S\in\mathbb{Z}`` and
+``M_S'=2M_S\in\mathbb{Z}``, as in the original article.
+
+```@docs
+AtomicLevels.xu_terms
+AtomicLevels.Xu.X
+AtomicLevels.Xu.A
+AtomicLevels.Xu.f
+```
```

Comment:
No, not really, but I wanted to somewhat complicated procedure documented/properly referenced, since previously the formulas were in a big comment block which only was readable if your editor supports inline LaTeX rendering (I'm only aware of Emacs). We should probably mark these functions as "internals".

Reply:
Makes sense. It is nice to have it in the manual. I'd say leave it as is here and let's revisit the question of how to mark what is and is not public API later.